docs/auto-setup.adoc

```diff
@@ -0,0 +1,111 @@
+= Automatic setup on virtual machine
+
+== Prerequisites
+
+You need to install:
+
+- https://www.packer.io/downloads[Packer]
+- https://www.vagrantup.com/downloads[Vagrant]
+- https://www.virtualbox.org/wiki/Downloads[VirtualBox]
+
+[TIP]
+VirtualBox installation may fail on macOS Catalina. If it's your case, please look
+at https://apple.stackexchange.com/questions/372492/virtualbox-installation-failed-on-macos-catalina[this issue].
+
+== Installing the system
+
+First, go to the `deployments/local-setup-environment` directory and build the environment image:
+```
+packer build -only=vagrant packerfile.json
+```
+An output directory named `vagrant-box` containing `package.box` file should be created.
+
+Then, switch the directory to `deployments/local-setup-instance` and import
+the image:
+```
+vagrant box add --force --name=local-setup-environment ./../local-setup-environment/vagrant-box/package.box
+```
+Finally, you can run the virtual machine by doing:
+```
+vagrant up
+```
+A new virtual machine will be created and provisioned accordingly. When
+this command terminates successfully, the machine should be ready to work
+and have all the auxiliary software like Geth, Bitcoin Core, ElectrumX
+and Core/ECDSA clients running.
```

Comment:
What versions of `Core/ECDSA` clients are there? Those referenced by git modules? I think it's worth being explicit about it.

Reply:
Yeah, when the `./install-repositories.sh` is triggered, it clones recent master for each submodule. I'll leave some details here.